integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -160,6 +168,42 @@ public void shouldPassThroughRecordUnchanged() throws Exception {
 
     }
 
+    @Test
+    public void shouldOfferPrometheusMetricsScrapeEndpoint() throws Exception {
+        String proxyAddress = "localhost:9192";
+
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder().testInfo(testInfo).build())) {
+            cluster.start();
+
+            String config = """
+                    proxy:
+                      address: %s
+                    metrics:
+                      host: localhost
+                      port: 9193
+                    clusters:
+                      demo:
+                        bootstrap_servers: %s
+                    filters:
+                    - type: ApiVersions
+                    - type: BrokerAddress
+                    """.formatted(proxyAddress, cluster.getBootstrapServers());
+
+            var proxy = startProxy(config);
+
+            String counter_name = "test_metric_" + new Random().nextLong() + "_total";
```

Comment:
d'oh, when it's negative the - symbol is translated to an underscore by micrometer and causes the test to fail (also the exception causes the proxy to not shutdown, a problem for another PR)